backend/migrate.sh

```diff
@@ -31,24 +31,36 @@ generate_migration() {
         exit 1
     fi
     echo "🛠  Generating new migration: $1"
-    poetry run alembic revision --autogenerate -m "$1"
-    echo "✅  New migration created."
+    if poetry run alembic revision --autogenerate -m "$1"; then
```

Comment:
This is a nice addition ty

Reply:
You're welcome! Thanks for reviewing my PR!